paymentsheet/src/test/java/com/stripe/android/paymentsheet/verticalmode/DefaultPaymentMethodVerticalLayoutInteractorTest.kt

```diff
@@ -166,6 +211,26 @@ class DefaultPaymentMethodVerticalLayoutInteractorTest {
         }
     }
 
+    @Test
+    fun handleViewAction_PaymentMethodSelected_doesNotChangeDisplayedSavedPaymentMethod() {
```

Comment:
This is surprising, why doesn't this change?

Reply:
in this particular test, an LPM is selected, so the saved payment method that is displayed doesn't change, though it wouldn't be shown as "selected" anymore. 

The alternative to this is that when you click on cashapp, the saved card that's displayed at the top of the list would change to your first saved PM, which seems like surprising + weird behavior

Reply:
I see, maybe something that would help me better understand it is to add an assert for the selected index.

Reply:
updated